util/translate_timeline.ts

```diff
@@ -101,5 +115,5 @@ const run = async (args) => {
   }
 };
 
-const args = parser.parseArgs();
+const args = parser.parseArgs() as { locale: Lang; timeline: string };
```

Comment:
I guess there's no way around the `as` here.

Reply:
Not as far as I could tell